### PR TITLE
Add actions to run smoke tests on staging and production.

### PR DESCRIPTION
**Pull request by tim-schilling**
Production is configured to only run on request since it does delete a blog and I'm not sure if we have backups yet.

### CODE REVIEW

File: .github/workflows/main_end_to_end_tests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Staging smoke test
+
+# Only trigger, when the build workflow succeeded
```
Comment (sabderemane) [inline .github/workflows/main_end_to_end_tests.yml:3]:
```suggestion
# Manual trigger, when the build workflow succeeded
```
This is manual only for main or did I miss something?

File: .github/workflows/main_end_to_end_tests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Staging smoke test
```
Comment (sabderemane) [inline .github/workflows/main_end_to_end_tests.yml:1]:
```suggestion
name: Main smoke tests
```

File: .github/workflows/develop_end_to_end_tests.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: Staging smoke test
```
Comment (sabderemane) [inline .github/workflows/develop_end_to_end_tests.yml:1]:
```suggestion
name: Staging smoke tests
```

**Review by sabderemane — COMMENTED** [review]
Nice, just few wording comments

File: .github/workflows/main_end_to_end_tests.yml (hunk 1 above)
Comment (tim-schilling) [reply .github/workflows/main_end_to_end_tests.yml:3]: You're 100% right. I copy/pasted, adjusted and forgot to clean up.

**Comment by tim-schilling** [comment]
Before this can be merged, we need to create the test user on the staging environment to confirm this will work.

**Review by sabderemane — APPROVED** [review]

**Comment by tim-schilling** [comment]
This doesn't seem to be a priority any more

**Pull request closed without merging**